modules/osd-aws-privatelink-about.adoc

```diff
@@ -0,0 +1,7 @@
+[id="osd-aws-privatelink-about.adoc_{context}"]
+= Understanding PrivateLink
+
+A {product-title} cluster can be created without any requirements on public subnets or public internet access. In this configuration, Red Hat uses AWS PrivateLink to manage and monitor a cluster in order to avoid any public ingress or egress network traffic.
```

Comment:
Suggest changing "Red Hat uses AWS PrivateLink to manage..." to "Red Hat uses AWS PrivateLink ("rivateLink)to manage..."